protocol/x/clob/types/keys.go

```diff
@@ -18,94 +18,91 @@ const (
 const (
 	// ProcessProposerMatchesEventsKey is the key to retrieve information about how to update
 	// memclob state based on the latest block.
-	ProcessProposerMatchesEventsKey = "process_proposer_matches_events"
+	ProcessProposerMatchesEventsKey = "ProposerEvents"
 
 	// SubaccountLiquidationInfoKeyPrefix is the prefix to retrieve the liquidation information
 	// for a subaccount within the last block.
-	SubaccountLiquidationInfoKeyPrefix = "subaccount_liquidation_info/"
+	SubaccountLiquidationInfoKeyPrefix = "SaLiqInfo/"
 
 	// LiquidationsConfigKey is the key to retrieve the liquidations config.
-	LiquidationsConfigKey = "liquidations_config"
+	LiquidationsConfigKey = "LiqCfg"
 
 	// EquityTierLimitConfigKey is the key to retrieve the equity tier limit configuration.
-	EquityTierLimitConfigKey = "equity_tier_limit_config"
+	EquityTierLimitConfigKey = "EqTierLimCfg"
```

Comment:
Suggestion: this is probably enough honestly
```suggestion
	EquityTierLimitConfigKey = "EqTierCfg"
```